clarify which server uses the shared secret too (#925)
hasura/graphql-engine: docs/graphql/manual/auth/jwt.rst

     "key": "3EK6FD+o0+c7tzBNVfjpMkNDi2yARAAKzQlk8O2IKoxQu4nF7EdAh8s3TwpHwrdWT6R"
   }

> The ``key`` is the actual shared secret, which is used by your Hasura graphql-engine server as well.

Reviewer: ```suggestion
The ``key`` is the actual shared secret, which is also used by your auth server.
```
